### DISCUSSION TITLE
mspider/troubleshoot/mesh-space-cannot-unbind

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2927] [Announcements]
# mspider/troubleshoot/mesh-space-cannot-unbind

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/troubleshoot/mesh-space-cannot-unbind.html

**Comment by hanxiaop** [comment]
这个现在已经过时了

**Comment by nicole-lihui** [comment]
`找到这个 mesh-hosted-apiserver` 这里需要描述是托管集群， 如果是专有网格模式，直接在相应集群操作

**Reply by nicole-lihui** [reply]
kubeadm init phase kubeconfig admin
alias kubectl="kubectl --kubeconfig=/etc/kubernetes/admin.conf --insurce-skip-tls-verify"

现在不需要了，hosted-api-server 内置了 kubeconfig

**Comment by windsonsea** [comment]
Fixed in #3506
